### PR TITLE
Feat/fusion framework/person components cookbook

### PR DESCRIPTION
**Pull request by AndrejNikolicEq**
## Why

- Working example of PersonCard and PersonAvatar
- Profile image from Person resolver

closes:

### Check off the following:
- [x] Confirm that I checked changes to branch which I am merging into.
  - _I have validated included files_
  - _My code does not generate new linting warnings_
  - _My PR is not a duplicate, [check existing pr`s](https://github.com/equinor/fusion-framework/pulls)_
- [x] Confirm that the I have completed the [self-review checklist](https://github.com/equinor/fusion-framework/blob/main/contributing/self-review.md).

- [x] Confirm that my changes meet our [code of conduct](https://github.com/equinor/fusion-framework/blob/main/CODE_OF_CONDUCT.md).

#### Conditional
- [ ] Confirm project selected and category, actual, iteration are set
- [ ] Confirm Milestone selected _(if any)_

### CODE REVIEW

File: cookbooks/app-react-person-component/src/App.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        '49132c24-6ea4-41fe-8221-112f314573f0',
+        'cbc6480d-12c1-467e-b0b8-cfbb22612daa',
+        'c81d9c4f-c882-4631-b0da-45e9ab9039c7',
+        '3268952b-f566-49ed-a2b4-6feda8c0d5ef',
+        '9716b649-5a4b-4470-9e0c-622742dc2935',
+        '1ea5f203-c1ad-4893-bdea-4fadd95455e4',
+    ];
+
+    return (
+        <StrictMode>
```
Comment (odinr) [inline cookbooks/app-react-person-component/src/App.tsx:38]: Remove

File: cookbooks/app-react-person-component/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                gap: theme.spacing.comfortable.medium.getVariable('padding'),
+                justifyContent: 'center',
+            },
+            card: {
+                width: '100%',
+                maxWidth: '400px',
+            },
+        }),
+    { name: 'person' }
+);
```
Comment (odinr) [inline cookbooks/app-react-person-component/src/App.tsx:23]: Change to StyledComponents and import EDS token

File: cookbooks/app-react-person-component/CHANGELOG.md
Comment (odinr) [inline cookbooks/app-react-person-component/CHANGELOG.md:1]: Clear content

**Review by odinr — CHANGES_REQUESTED** [review]
Fix comments and add change set

**Comment by AndrejNikolicEq** [comment]
done in #1107

**Pull request closed without merging**